frontend/tests/utils/wmgUtils.ts

```diff
@@ -104,22 +104,23 @@ export async function searchAndAddTissue(page: Page, tissueName: string) {
     async () => {
       await page.getByTestId(ADD_TISSUE_ID).getByRole("button").first().click();
       await page.getByRole("tooltip").waitFor();
+      await page.getByRole("tooltip").getByText(tissueName).first().click();
```

Comment:
fix flake: sometimes we select tissue too fast that it's not registered in the DOM. So we select and assert the tag does show up in a `tryUntil()`